packages/ui-file-drop/src/FileDrop/index.tsx

```diff
@@ -113,6 +121,12 @@ class FileDrop extends Component<FileDropProps, FileDropState> {
     ;(this.props as any).makeStyles(this.makeStyleProps())
```

Comment:
You should fix these too: `this.props.makeStyles?.`. Generally there should be no `any`s unless really necessary.